src/HotChocolate/Core/test/Types.Tests/Types/Relay/__snapshots__/IdAttributeTests.Id_On_Objects_Given_Nulls.snap

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "result": "{\n  \"data\": {\n    \"foo\": {\n      \"someId\": \"QmFyCmQx\",\n      \"someNullableId\": null,\n      \"someIds\": [\n        \"QmFyCmkx\"\n      ],\n      \"someNullableIds\": [\n        null,\n        \"QmFyCmkx\"\n      ]\n    }\n  }\n}",
```

Comment:
Sends and receives some nullable fields too now; but in this test also passes through `null`s in contrast to test above.